sample-code/spring-app/src/test/java/com/sap/ai/sdk/app/controllers/OrchestrationTest.java

```diff
@@ -76,7 +75,7 @@ void testTemplate() {
 
   @Test
   void testLenientContentFilter() {
-    var response = controller.filter(AzureFilterThreshold.ALLOW_SAFE_LOW_MEDIUM);
+    var response = controller.filter(AzureFilterThreshold.ALLOW_ALL);
```

Comment:
flaky tests

Reply:
They just failed for me with finish reason content filter instead of stop

Reply:
Can't reproduce it so far. Also here, please revert, if anything a change should be done in a separate PR.